fbinternal/cloud/go/services/testcontroller/store/testcontroller_store.go

```diff
@@ -159,12 +159,12 @@ func (s *TestControllerStore) isMinimumWaitTimeElapsed(timestamp int64, minimumW
 }
 
 func (s *TestControllerStore) get(networkID string) (string, int64, error) {
-	typeAndKey := storage.TK{
+	TK := storage.TK{
```

Comment:
Should be lower case since it's local
```suggestion
	tk := storage.TK{
```